packages/imperative/src/config/cmd/secure/secure.handler.ts

```diff
@@ -57,29 +57,24 @@ export default class SecureHandler implements ICommandHandler {
         }
 
         // Prompt for values designated as secure
-        let authTokenProp: string;
         for (const propName of secureProps) {
-            if (authTokenProp == null && propName.endsWith(".tokenValue")) {
-                authTokenProp = propName;
-                continue;
-            }
-
-            let propValue = await params.response.console.prompt(`Enter ${propName} - blank to skip: `, { hideText: true });
-
-            // Save the value in the config securely
-            if (propValue) {
-                propValue = coercePropValue(propValue, ConfigSchema.findPropertyType(propName, config.properties));
-                config.set(propName, propValue, { secure: true });
-            }
-        }
-
-        if (authTokenProp != null) {
-            const propValue = await this.handlePromptForAuthToken(config, authTokenProp) ||
-                await params.response.console.prompt(`Enter ${authTokenProp} - blank to skip: `, {hideText: true});
-
-            // Save the value in the config securely
-            if (propValue) {
-                config.set(authTokenProp, propValue, { secure: true });
+            if (propName.endsWith(".tokenValue")) {
+                params.response.console.log(`Processing secure properties for profile: ${config.api.profiles.getProfileNameFromPath(propName)}`);
+                const propValue = await this.handlePromptForAuthToken(config, propName) ||
+                    await params.response.console.prompt(`Enter ${propName} - blank to skip: `, {hideText: true});
+
+                // Save the value in the config securely
+                if (propValue) {
+                    config.set(propName, propValue, { secure: true });
+                }
+            } else {
+                let propValue = await params.response.console.prompt(`Enter ${propName} - blank to skip: `, { hideText: true });
```

Comment:
The phrase "blank to skip" has confused at least one consumer. After several failed attempts to run the secure command (due to other reasons), the consumer began to wonder if they were supposed to literally enter a blank space to skip the entry for a given secure property.

While we are making these changes, we could consider being more explicit in our prompt:

    Enter ${propName} - Press ENTER to skip

This might not be the only place that we use the phrase "blank to skip". Even so, I would choose to make an incremental (but inconsistent) improvement over maintaining the consistency of a confusing prompt.

Reply:
I made them all consistent 😋 
I don't believe this is a breaking change as this messages only show up in interactive mode 😋